finger-exercises/redux/src/redux/book/reducer.js

```diff
@@ -6,18 +6,29 @@ const initialState = {
   originalData: []
 };
 
+const filteredBooks = (arr, searchText) => arr.filter(item => item.name.toLowerCase().includes(searchText));
```

Comment:
These functions `filteredBooks`, `sumQuantity` and `removeItem` should be in a different file.
May be `utils.js`?